### ISSUE TITLE
Add a11y e2e tests to ASM

### ISSUE THREAD

**Issue by kacperknapik** [SAP/cloud-commerce-spartacus-storefront#6314]
Master ticket: #6227

According to https://wiki.hybris.com/display/spar/How+to+implement+accessibility+tests+and+fixes please add accessibility e2e tests for ASM

**Comment by KateChuen** [comment]
Running the test locally, pointing to dev17 backend. 

the tabbing-order and the asm tests are failing consistently.

Closing this one, since we have https://github.com/SAP/spartacus/issues/7020 to tackle the tabbing-order test.
and https://github.com/SAP/spartacus/issues/7063 as a master ticket to watch the occasional failure of the asm test.
